src/app/helptext/jails/jails-edit.ts

```diff
@@ -16,6 +16,9 @@ dhcp_tooltip: T('Set to start the jail with the Dynamic Host \
 Configuration Protocol enabled. <b>VNET</b> and \
 <b>Berkeley Packet Filter</b> must also be enabled.'),
 
+nat_placeholder: T('NAT'),
+nat_tooltip: T(''),
```

Comment:
Network Address Translation (NAT). Transforms local network IP addresses into a single IP address. Set when the jail will share a single connection to the Internet with other systems on the network.